.github/ISSUE_TEMPLATE/test-plan.md

```diff
@@ -0,0 +1,62 @@
+## Test plan
+
+## Feature
+
+Name:
+
+Description:
+
+Group: CX / Infra / MDM / Website
+
+## DRIs
+
+Engineer:
+
+QA: 
+
+## Changes
+
+Breaking Changes: Y/N
+
+Clean database: Y/N
+
+## Risk assessment
+
+Risk level: Low / Medium / High
+
+Risk area: 
```

Comment:
Not sure we're ready for this complexity yet.  Maybe crawl before we walk (i.e. minimally viable process first)